ui.js

```diff
@@ -60,7 +55,7 @@ async function main() {
   .option('interactive', {
     alias: 'i',
     describe: 'Whether to run in interactive mode',
-    default: true,
+    default: false,
```

Comment:
Technically speaking if we do `getTaskInput` this is already considered interactive.
So to be logically complete if the task wasn't provided on the command line, the non-interactive mode should fail.
https://github.com/fjrdomingues/autopilot/pull/53/files#diff-f2e0e6d1424d62935e6aac986cdca3b8bfc33333b452b3015494510fecced163R52

Generally, I think we are still in the broken alpha stage and this interactive mode helps people see what's going on, break early or retry.
(Ultimately I wanted to also offer a restart to put a new updated task, but one logic jump that I stopped at for the last patch)

At least documenting Interactive mode in README.md before committing this PR is a good idea.
while we call it interactive it's also debug, which while both we and the users are figuring how to use this, seems useful to me

but I'm not blocking 👍

Reply:
Thanks for the input. I updated the README file